charts/code-server/templates/deployment.yaml

```diff
@@ -74,11 +74,7 @@ spec:
         {{- end }}
           volumeMounts:
           - name: data
-            mountPath: /home/coder/project
-            subPath: project
-          - name: data
-            mountPath: /home/coder/.local/share/code-server
-            subPath: code-server
+            mountPath: /home/coder
```

Comment:
This is to combine both mounts in a single mount. It is useful in mounting the entire home inside the pod. For example:
```
persistence:
  hostPath: /home/hammady
securityContext:
  enabled: true
  # get user id and group using the `id` command
  fsGroup: ??
  runAsUser: ??
volumePermissions:
  enabled: false
```
The benefit of this is that developers can mount their `.bashrc`, other dot files, local binaries (e.g. `kubectl`, `helm`, ...) inside the pod and starting using them right away.

Reply:
If they don't mount the entire home, they have to reinstall all binaries and do all edits to the dot files every time the pod is restarted.

Reply:
@Matthew-Beckett what do you think?